molgenis-web/src/main/java/org/molgenis/web/PluginInterceptor.java

```diff
@@ -78,6 +78,7 @@ public void postHandle(HttpServletRequest request, HttpServletResponse response,
 			modelAndView.addObject(PluginAttributes.KEY_AUTHENTICATED, SecurityUtils.currentUserIsAuthenticated());
 			modelAndView.addObject(PluginAttributes.KEY_PLUGIN_ID_WITH_QUERY_STRING,
 					getPluginIdWithQueryString(request, pluginId));
+			modelAndView.addObject(PluginAttributes.KEY_PLUGIN_DESCRIPTION, molgenisPlugin.getDescription(pluginId));
```

Comment:
How about using one key - value that contains the plugin id and description (and possibly label, see previous remark) so we don't have to update the interceptor when we change Plugin attributes in the future.